DEVELOPMENT.md

```diff
@@ -70,12 +70,12 @@ When using or developing the authenticated version of ASReview, extra steps
 are needed to configure the application.
 
 Create an authentication config file as instructed in [Authentication]
-(#Authentication). Set the environment variable `FLASK_CONFIGFILE` to the
+(#Authentication). Set the environment variable `ASREVIEW_LAB_CONFIG_PATH` to the
```

Comment:
`FLASK_CONFIGFILE` didn't work for me, so I updated the docs, but maybe it's just me. :)

Reply:
You are right! `ASREVIEW_LAB_CONFIG_PATH ` it is.